modules/testsuite/pom.xml

```diff
@@ -659,12 +678,13 @@
           <scope>test</scope>
         </dependency>
       </dependencies>
+
     </profile>
     <!--
-    Descr: Test with the latest WFLY27 upstream version
+    Descr: Test with the latest WFLY28 upstream version
     -->
     <profile>
-      <id>wildfly27dev</id>
+      <id>wildfly28dev</id>
       <properties>
         <additionalGroovyScriptSuffix>-nolegacy</additionalGroovyScriptSuffix>
```

Comment:
this can be removed, it is not used anywhere anymore

Reply:
Fixed. Thanks @jbliznak